packages/repository/src/__tests__/unit/decorator/model-and-relation.decorator.unit.ts

```diff
@@ -132,6 +132,41 @@ describe('model decorator', () => {
     recentOrders?: Order[];
   }
 
+  it('hides a property defined as hidden', () => {
+    @model()
+    class Client extends Entity {
+      @property()
+      name: string;
+      @property({hidden: true})
+      password: string;
+
+      constructor(data: Partial<Client>) {
+        super(data);
+      }
+    }
+    const client = new Client({
+      name: 'name',
+      password: 'password',
+    });
+    expect(Client.definition.settings.hiddenProperties).to.containEql(
+      'password',
+    );
+    expect(client.toJSON()).to.eql({
+      name: 'name',
+    });
+  });
+  it('Throw error if desing type is not provided', () => {
```

Comment:
We write test names to read as English sentences. Also please add an empty line before the test case to serve as a visual delimiter.

```suggestion

  it('throws error if design type is not provided', () => {
```